notebooks/submission_tutorial/1_sensorium_submission.ipynb

```diff
@@ -0,0 +1,495 @@
+{
```

Comment:
<p>".. normalized by its own STD" -&gt; ".. divided by its own STD (which is computed across all images in the train set)".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/sensorium/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>fixed</p>